### PR TITLE
Small fixes 1

### PR DESCRIPTION
**Pull request by dortheimer**

### CODE REVIEW

File: src/components/Wrapper/Wrapper.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import Navigation from "../Navigation";
+import Footer from "../Footer";
+
+export default function Wrapper(props) {
+    const { me, children } = props;
+    return (
+        <Fragment>
+            <Navigation me={me} />
+            {!children ? (
+                <div style={{ textAlign: "center" }}>
```
Comment (GalGend) [inline src/components/Wrapper/Wrapper.jsx]: why style not in class

File: src/scenes/About.jsx [hunk 2]
```diff
@@ -1,118 +1,159 @@
...
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="col-6 col-lg-4">
+                                    <div class="m-3 text-center">
+                                        <img
+                                            class="img-fluid"
+                                            src={adi}
+                                            alt="עדי קויש"
```
Comment (GalGend) [inline src/scenes/About.jsx]: why not remove them on the way

File: src/scenes/Home.jsx [hunk 3]
```diff
@@ -1,61 +1,58 @@
...
-                מערכת ההתראות שלנו מאגדת במקום אחד את כל התכניות בסמכות מקומית
-                ומחוזית ותשלח לכם התראה למייל בזמן אמת, בכל פעם שתופקד תכנית
-                חדשה בסמכות מקומית או כשתקלט במערכת תכנית חדשה בסמכות מחוזית
-                באזור מגוריכם או בכל אזור אחר שתבחרו! כך תוכלו לעקוב בקלות אחרי
-                השינויים בסביבת המגורים שלכם.
-              </p>
+        <Wrapper me={me}>
+            <div
+                className="d-none d-md-block"
+                style={{
```
Comment (GalGend) [inline src/scenes/Home.jsx]: why not in style

Comment (dortheimer) [reply src/scenes/Home.jsx]: Since it doesn't make sense to create a call for single use.

File: src/scenes/About.jsx (hunk 2 above)
Comment (dortheimer) [reply src/scenes/About.jsx]: @eyalmigd  can we remove adi?

File: src/components/Wrapper/Wrapper.jsx (hunk 1 above)
Comment (dortheimer) [reply src/components/Wrapper/Wrapper.jsx]: I think for a single use there is no need to create a class in React.

**Pull request merged**